worker-workflow/src/main/java/com/github/cafdataprocessing/workflow/WorkflowManager.java

```diff
@@ -37,21 +37,29 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import org.apache.commons.io.FileUtils;
 
 public class WorkflowManager {
 
     private final Map<String, Workflow> workflows;
     private final DataStore dataStore;
+    private final String contextScriptFilePath;
```

Comment:
You don't need to store this.  Just pass it into the `getWorkflows()` function as a parameter.